### PR TITLE
Add support for checking if zip already uploaded to browserstack.

### PR DESCRIPTION
**Pull request by sparsh00**
Added support to check if zip already uploaded by checking md5 of the zip files.

### CODE REVIEW

File: bin/helpers/constants.js [hunk 1]
```diff
@@ -145,6 +147,21 @@ const filesToIgnoreWhileUploading = [
   '.yarn/**',
 ];
 
+const readDirOptions = {
+  cwd: '.',
```
Comment (karanshah-browserstack) [inline bin/helpers/constants.js:151]: Why are we not using cypressFolderPath as CWD? We have the user's who have their cypress.json and CWD different.

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: bin/helpers/constants.js (hunk 1 above)
Comment (sparsh00) [reply bin/helpers/constants.js:151]: Will be using that only https://github.com/browserstack/browserstack-cypress-cli/pull/148/files#diff-e4adc3c50732024d070d2df81b167010ada80e39e52edfc8f3f48b0e9b44a501R18 https://github.com/browserstack/browserstack-cypress-cli/pull/148/files#diff-a91b9660eaaccd540cc602da6acb2b2f1bcad2dc63d25de89bd569463295d57eR20 this is just for fallback.

File: bin/helpers/checkUploaded.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+'use strict';
+const request = require('request');
+
+const crypto = require('crypto'),
```
Comment (shashikdm) [inline bin/helpers/checkUploaded.js:4]: this is a new dependency. we may need to add this in `package.json`

Comment (shashikdm) [reply bin/helpers/checkUploaded.js:4]: ignore. its part of node js standard library

**Review by shashikdm — APPROVED** [review]

**Review by karanshah-browserstack — APPROVED** [review]

**Review by shashikdm — APPROVED** [review]

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**